examples/datacenter.rb

```diff
@@ -9,6 +9,8 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+# NOTE: It's necessary there is a Rack with name 'Rack_1' added before
```

Comment:
Avoid "It's", *it is* should be used in these cases.
Suggested rewording:

*NOTE: This example requires a Rack named "Rack_1" in the appliance.*  

~~or something similar